src/config/2020.json

```diff
@@ -757,6 +757,14 @@
       "github": "thefoxis",
       "twitter": "fox"
     },
+    "tymosh": {
+      "name": "Kate Tymoshkina",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
+      "github": "tymosh"
+    },
```

Comment:
Same here for 2020

Reply:
Please, add LinkedIn account: https://www.linkedin.com/in/tymosh/

Reply:
```suggestion
    "tymosh": {
      "name": "Kate Tymoshkina",
      "teams": [
        "translators"
      ],
      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
      "github": "tymosh",
      "linkedin": "tymosh"
    },
```